test_app/settings.py

```diff
@@ -23,7 +23,7 @@
 
```

Comment:
`ImproperlyConfiguredError` would only be raised if your backend was being used correct? It's not needed by default